lib/test_prof/event_prof/profiler.rb

```diff
@@ -73,13 +73,18 @@ def example_finished(id)
       end
 
       def results
-        {
+        results = {
           groups: @groups.to_a
         }.tap do |data|
+          data[:groups_total_time] = data[:groups].reduce(0) do |sum, group|
```

Comment:
@palkan could you give some feedback on this approach? I think it's right, but I would like to be sure.